### DISCUSSION TITLE
[I donno man] DB Folder to the mooooon!

### DISCUSSION THREAD

**Discussion by rohanbatraio** [RafaelGB/obsidian-db-folder#473] [Show and tell]
![2022-10-07-221502_2732x768_scrot](https://user-images.githubusercontent.com/113782197/194605294-b9aafc54-038e-472a-bc79-c6e54dc29e8c.png)

:laughing:  I just don't know what to say, while i am working on an upcoming project i just reached this count on my database count. What this means is i have 527 databases. 

AND COUNTING .... 
I don't know where this will lead me :rofl: but it's fun, if anyone is wondering what this is all about, OBSIDIAN OCTOBER EVENT.

**Comment by RafaelGB** [comment]
wow! too much ddbbs =P You really need the templates, now I can see why

I'm excited that you can get so much use out of it

**Reply by rohanbatraio** [reply]
Yes of course, you would see all kind of request coming from my end :rofl:

**Reply by rohanbatraio** [reply]
I am custom coding templates for now using Bash scripting :smiley:
